### PR TITLE
OCPQE-18079: Install OCP to a shared VPC, with credentialsMode "Manual" and Workload Identity

### PR DESCRIPTION
**Pull request by jianli-wei**
- add the workflow `cucushift-installer-rehearse-gcp-ipi-xpn-cco-manual-workload-identity` and the provision & deprovision chains
- add the steps `gcp-provision-iam-creds-xpn-roles` and `gcp-deprovision-iam-creds-xpn-roles`, to grant/remove the required roles to/from the related service-accounts in the GCP host project
- add periodic tests `gcp-ipi-xpn-workload-identity-amd-f28-destructive` & `gcp-ipi-xpn-workload-identity-arm-f28` in 4.14 & 4.15 & 4.16 multi nightly

### CODE REVIEW

**Comment by jianli-wei** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installation-nightly-4.15-gcp-ipi-xpn-workload-identity-f28

**Comment by jianli-wei** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installation-nightly-4.15-gcp-ipi-xpn-workload-identity-f28

**Comment by jianli-wei** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installation-nightly-4.15-gcp-ipi-xpn-workload-identity-f28

**Comment by jianli-wei** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installation-nightly-4.15-gcp-ipi-xpn-workload-identity-f28

**Comment by jianli-wei** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installation-nightly-4.15-gcp-ipi-xpn-workload-identity-f28

**Comment by jianli-wei** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installation-nightly-4.15-gcp-ipi-xpn-workload-identity-f28

**Comment by jianli-wei** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installation-nightly-4.15-gcp-ipi-xpn-workload-identity-f28

**Comment by jianli-wei** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installation-nightly-4.15-gcp-ipi-xpn-workload-identity-f28

**Comment by jianli-wei** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installation-nightly-4.15-gcp-ipi-xpn-workload-identity-f28

File: ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.14.yaml [hunk 1]
```diff
@@ -70,6 +70,13 @@ tests:
     test:
     - ref: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-aws-ipi-preserve-bootstrap-ign
+- as: gcp-ipi-xpn-workload-identity-f28
```
Comment (jianlinliu) [inline ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.14.yaml:73]: I think we also can move the job into the common ci like others.

**Comment by jianlinliu** [comment]
cc @liangxia to review.

**Pull request merged**